### ISSUE TITLE
What is est?

### ISSUE THREAD

**Issue by nsheff** [databio/logmuse#11]
logmuse is held in a file called `est.py`. What does this name mean? Can we name it something more understandable?

**Comment by vreuter** [comment]
Yeah, it'd be fine to change. I think I called it that as short for "establish" since it's home to the main setup functions. The only problem I'd see is breaking imports, but I think the package `__init__` and the `__all__` declarations are set up to expose at top level anything that's intended for import and use outside of the package itself, so ideally client code is importing from `logmuse` directly rather than the `est` module and would be safe.

**Comment by nsheff** [comment]
Ok thanks. when we get to this we should also consider: https://github.com/pepkit/looper/issues/224
